.ycm_extra_conf.py

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env python
-
+# -*- Mode: python; coding: utf-8; indent-tabs-mode: nil -*-
```

Comment:
hmm, this is for emacs, no? but why would emacs ever assume something else than utf-8?

a while back we dropped all emacs mode-lines from the project, in favour of dir-locals. Why not handle this the same way?

Reply:
I understand now that this is actually not for emacs but for python. However it appears to me this is unnecessary for python3:

https://www.python.org/dev/peps/pep-3120/

Hence I figure this commit can be dropped?

Reply:
Yeah, dropped.